src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -311,7 +344,7 @@ public Person generatePerson(int index, long personSeed) {
         }
         
         if (this.metrics != null) {
-          metrics.recordStats(person, time);
+          metrics.recordStats(person, time, modules);
```

Comment:
```diff
- metrics.recordStats(person, time);
+ metrics.recordStats(person, time, modules);
```
This doesn't exactly work, because the modules are being removed in the underlying collection in L318:

```java
iter.remove(); // this module has completed/terminated.
```

If you do this, only the modules left at the end (which could possibly be none except a few core modules) will be recorded -- you'll be missing a lot of data.

Reply:
I'll fix this after merging.